packages/web/src/core/utils/web3-connectors.ts

```diff
@@ -28,9 +28,9 @@ export const chainConfigs = {
     chainId: utils.hexValue(595),
     chainName: 'Mandala TC7',
     nativeCurrency: {
-      decimals: 18,
+      decimals: 12,
       name: 'Acala USD',
-      symbol: 'ACA'
+      symbol: 'aUSD'
     },
```

Comment:
I'm sorry, I saw @zamith's comment and didn't realize this was about the native currency. This was actually correct as ACA
The other two are the problem

Mandala's native currency is Acala. "Acala USD" is a second currency of the system, but not the main one

Reply:
@naps62 Take another look and merge it if ok.